### ISSUE TITLE
return entry['organism_name'].split(' ')[1] IndexError: list index out of range

### ISSUE THREAD

**Issue by tseemann** [kblin/ncbi-genome-download#24]
```
% ncbi-genome-download --parallel 4 -H -d -l complete viral

<snip>
DEBUG: Starting new HTTPS connection (1): ftp.ncbi.nlm.nih.gov
DEBUG: https://ftp.ncbi.nlm.nih.gov:443 "GET /genomes/all/GCF/000/838/005/GCF_000838005.1_ViralProj14104/md5checksums.txt HTTP/1.1" 200 649
INFO: Downloading record 'GCF_000847185.1'
DEBUG: Starting new HTTPS connection (1): ftp.ncbi.nlm.nih.gov
DEBUG: https://ftp.ncbi.nlm.nih.gov:443 "GET /genomes/all/GCF/000/847/185/GCF_000847185.1_ViralProj14595/md5checksums.txt HTTP/1.1" 200 649
INFO: Downloading record 'GCF_000883035.1'
DEBUG: Starting new HTTPS connection (1): ftp.ncbi.nlm.nih.gov
DEBUG: https://ftp.ncbi.nlm.nih.gov:443 "GET /genomes/all/GCF/000/883/035/GCF_000883035.1_ViralProj31249/md5checksums.txt HTTP/1.1" 200 649
INFO: Downloading record 'GCF_000864565.1'
Traceback (most recent call last):
  File "/home/linuxbrew/.linuxbrew/bin/ncbi-genome-download", line 11, in <module>
    sys.exit(main())
  File "/home/linuxbrew/.linuxbrew/opt/python3/lib/python3.5/site-packages/ncbi_genome_download/__main__.py", line 73, in main
    ret = ncbi_genome_download.download(args)
  File "/home/linuxbrew/.linuxbrew/opt/python3/lib/python3.5/site-packages/ncbi_genome_download/core.py", line 60, in download
    args.taxid, args.human_readable, args.parallel)
  File "/home/linuxbrew/.linuxbrew/opt/python3/lib/python3.5/site-packages/ncbi_genome_download/core.py", line 91, in _download
    download_jobs.extend(download_entry(entry, section, domain, output, file_format, human_readable))
  File "/home/linuxbrew/.linuxbrew/opt/python3/lib/python3.5/site-packages/ncbi_genome_download/core.py", line 125, in download_entry
    symlink_path = create_readable_dir(entry, section, domain, output)
  File "/home/linuxbrew/.linuxbrew/opt/python3/lib/python3.5/site-packages/ncbi_genome_download/core.py", line 169, in create_readable_dir
    get_species_label(entry),
  File "/home/linuxbrew/.linuxbrew/opt/python3/lib/python3.5/site-packages/ncbi_genome_download/core.py", line 295, in get_species_label
    return entry['organism_name'].split(' ')[1]
IndexError: list index out of range
```

**Comment by kblin** [comment]
Trying to reproduce this locally. On a first guess, the viral summary file is structured differently, breaking the parser.

**Comment by kblin** [comment]
Yeah. Basically organism_name in the viral assembly summary file is completely unstructured and unusable, but not in the part of the file that I picked for the unit tests.

**Comment by kblin** [comment]
Should be fixed by 07b61f5, please reopen if it's still an issue.

**Comment by kblin** [comment]
Reopening issues where fix is not shipped yet.

**Comment by kblin** [comment]
Shipped in 0.2.3 release.
